src/ewig/buffer.cpp

```diff
@@ -202,7 +202,7 @@ line get_line(const text& txt, index row)
 
 index line_length(const line& ln)
 {
-    return utf8::unchecked::distance(ln.begin(), ln.end());
+    return utf8::unchecked::distance(std::begin(ln), std::end(ln));
```

Comment:
What was the error here? This should not be necessary (`line` is an `immer::flex_vector`, which supports `.begin()` and `.end()`)

Reply:
Yeah, this wasn't actually necessary. I just forgot to revert it.